src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -194,6 +194,7 @@ private static ScriptFunc ClearRequireCacheFunc
             object functionResult = await function(scriptExecutionContext);
 
             await ProcessOutputBindingsAsync(_outputBindings, input, context.Binder, bindingData, scriptExecutionContext, functionResult);
+            return null;
```

Comment:
>null [](start = 19, length = 4)

Do you plan on adding support for Node?  Is that simply a matter of returning functionResult here or would we also need to change the behavior of ProcessOutputBindingsAsync?

Reply:
I wonder this as well - is this change not applicable to node & other script types?

Reply:
Node is totally different work. I filed https://github.com/Azure/azure-webjobs-sdk-script/issues/1921 to track that.